<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeWindow</class>
 <widget class="QWidget" name="welcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to {solar}</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#background{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0397727 rgba(183, 66, 0, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>-110</y>
      <width>1151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>40</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Welcome to                  </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="uiScriptBasedSpammer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Script-based Spammer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uiUniqueSpammer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>Unique Spammer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 119, 121);</string>
    </property>
    <property name="text">
     <string>About solar</string>
    </property>
   </widget>
   <widget class="QLabel" name="solarLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>-110</y>
      <width>1151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Emoji</family>
      <pointsize>40</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>           {solar}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/icon.qrc"/>
 </resources>
 <connections/>
</ui>
